.travis.yml

```diff
@@ -4,3 +4,11 @@ node_js:
   - "4"
   - "0.12"
   - "0.10"
+
+script:
+  nv=$(node -v);
+  if [[ $nv == v6* ]] ; then
```

Comment:
As you pointed out, this is quite fragile. Can you use [check-node-version](https://www.npmjs.com/package/check-node-version) to check for Node >= 6?